src/lib/cli.ts

```diff
@@ -25,10 +25,10 @@ import { loadJSONFile } from './utils/misc';
 import * as logger from './utils/logging';
 import { cutString } from './utils/misc';
 import { options } from './ui/options';
+import { initSonarrc } from './ui/generator';
```

Comment:
I was thinking about moving `rule-generator` under `ui` as well. Maybe we should rename that folder to `cli` too.